stablehlo/dialect/VhloBase.td

```diff
@@ -59,43 +59,6 @@ def VHLO_VersionedTypeInterface : TypeInterface<"VersionedTypeInterface"> {
   ];
 }
 
-//===----------------------------------------------------------------------===//
-// VHLO Type Versioning
-//===----------------------------------------------------------------------===//
-
-class VHLO_TypeDef<string name,
-                   string minVersion = "0.3.0",
-                   string maxVersion = "current">
-  : TypeDef<VHLO_Dialect, name, [VHLO_VersionedTypeInterface]> {
-  let extraClassDeclaration = [{
-    mlir::vhlo::Version getMinVersion() {
-      auto version = mlir::vhlo::Version::fromString("}] #  minVersion # [{");
-      if (failed(version)) llvm_unreachable("invalid version }] # minVersion # [{ in }] # name # [{");
-      return *version;
-    }
-    mlir::vhlo::Version getMaxVersion() {
-      if (!strcmp("}] # maxVersion # [{", "current")) return VhloDialect::getCurrentVersion();
-      auto version = mlir::vhlo::Version::fromString("}] #  maxVersion # [{");
-      if (failed(version)) llvm_unreachable("invalid version }] # maxVersion # [{ in }] # name # [{");
-      return *version;
-    }
-  }];
-}
-
-//===----------------------------------------------------------------------===//
-// VHLO Type Definitions.
-//===----------------------------------------------------------------------===//
-
-// VHLO is intended to represent the layout only, as such uses AnyType everywhere.
-def VHLO_AnyType : AnyTypeOf<[AnyType]>;
-def VHLO_AnyAttr : AnyAttrOf<[AnyAttr]>;
-def VHLO_AnyRegion : Region<CPred<"true">, "any region">;
```

Comment:
I would we imagine we want to keep these three defs in this file, given that they are not TypeDefs or related to TypeDefs?

Reply:
I'll leave them in base for now. I think it would be a good sanity check to define:

```
def VHLO_AnyType : AnyTypeOf<[VHLO_TokenV1Type, VHLO_WrappedType, VHLO_IntegerV1Type, ...]>;
def VHLO_AnyAttr : AnyTypeOf<[VHLO_ChannelHandle, ....]>;
```

That said, this adds overhead to the process of adding a type/attr and may be better as a one-time experiment instead of something we submit.

Reply:
Moved back to VhloBase.td